### PR TITLE
filterable list updates and docs

### PR DESCRIPTION
**Pull request by nelsonpecora**
* added `active` styling to list items
* added `inputPlaceholder` option to override placeholder text
* added documentation about all the options you can pass in when creating filterable lists

### CODE REVIEW

File: services/filterable-list.js [hunk 1]
```diff
@@ -74,9 +74,28 @@ function appendAddButton(el, options) {
...
+
+  input.setAttribute('placeholder', text);
+}
+
+/**
  * create a filterable list
+ * note: add an `active` class to specific list items to apply the "active" styles to that item
  * @param {array} items
  * @param {object} options
+ * @param {function} options.click called with id when users click an item
```
Comment (byronhulcher) [inline services/filterable-list.js]: 🙌

**Comment by byronhulcher** [comment]
👍

**Comment by jonwinton** [comment]
🎁

**Comment by cruzanmo** [comment]
+1

**Pull request merged**